source/appModules/notepadPlusPlus.py

```diff
@@ -46,7 +46,9 @@ class NppEdit(ScintillaBase.Scintilla):
 	def _get_TextInfo(self):
 		if self.appModule.is64BitProcess:
 			appVerMajor, appVerMinor, *__ = self.appModule.productVersion.split(".")
-			if int(appVerMajor) >= 8 and int(appVerMinor) >= 3:
+			# appVerMinor could be either one digit (e.g. in 8.3), two digits (e.g. in 8.21), or even three (8.1.9.2)

```

Comment:
I'm not sure this comment is accurate. In the example in which the version is (8.1.9.2 `appVerMinor` would be one digit - 1.

Reply:
This is incorrect. appVerMinor would be 192.

In any case, your suggestion is way nicer.

Reply:
> This is incorrect. appVerMinor would be 192.

Sorry for insisting, but I've started doubting that I understand how `split` works  ;) so had to check and it turns out I'm right.
From the Python repl:
```
C:\Users\Lukasz>py
Python 3.8.10 (tags/v3.8.10:3d8993a, May  3 2021, 11:48:03) [MSC v.1928 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> appVer = "8.1.9.2"
>>> appVerMajor, appVerMinor, *__ = appVer.split(".")
>>> appVerMajor
'8'
>>> appVerMinor
'1'
>>> __
['9', '2']

```

Note that everything after the second dot is unpacked into a temporary `__` variable and is not used in the comparisons.